lxd/apparmor.go

```diff
@@ -22,6 +23,27 @@ var aaEnabled = false
 
 var aaPath = shared.VarPath("security", "apparmor")
 
+const NESTING_AA_PROFILE = `
+  pivot_root,
+  mount /var/lib/lxd/shmounts/ -> /var/lib/lxd/shmounts/,
+  mount none -> /var/lib/lxd/shmounts/,
+  mount fstype=proc -> /usr/lib/x86_64-linux-gnu/lxc/**,
+  mount fstype=sysfs -> /usr/lib/x86_64-linux-gnu/lxc/**,
+  mount options=(rw,bind),
+  mount options=(rw,rbind),
+  deny /dev/.lxd/proc/** rw,
+  deny /dev/.lxd/sys/** rw,
```

Comment:
I don't understand what these rules are doing here; if someone wants to pass something into lxd via mounts, why can't they pass proc/sys?